src/services/gui/gui_service.hpp

```diff
@@ -116,6 +117,7 @@ namespace big
 		            view::world,
 		            {
 		                {tabs::SPAWN_PED, {"GUI_TAB_SPAWN_PED", view::spawn_ped}},
+						{tabs::SQUAD_SPAWNER, {"Squad spawner", view::squad_spawner}},
```

Comment:
Weird indent

Reply:
Ye idk why this keeps happening even tho I press the format thingy on each edited file